csrc/cpu/comm/riscv64/shm.h

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0
+
+// DeepSpeed Team
+
+#include <riscv_vector.h>
+
+inline vfloat32m2_t cvt_bf16_to_fp32(vuint16m1_t src, size_t vl) __attribute__((target("arch=+v")));
+inline vuint16m1_t cvt_fp32_to_bf16(vfloat32m2_t src, size_t vl)
```

Comment:
when will `"arch=+v"` be used and when will `"arch=+v,+zvfh"` used?   I saw `+zvfh` relates to fp16 but cvt_fp32_to_bf16 seems not involving fp16.